docs/v0.1.4/api-reference/importer-props.mdx

```diff
@@ -121,7 +121,7 @@ Callback executed when the user has finished reviewing the import summary. This
 
 ---
 
-## indexDBConfig
+## persistenceConfig
```

Comment:
This shouldn't be changed in 0.1.4, since the field is called indexDBConfig in 0.1.4